### DISCUSSION TITLE
Hotkey Toggle button for using midi notes for system messages

### DISCUSSION THREAD

**Discussion by RK11111111111** [probonopd/MiniDexed#606] [Ideas]
I think it would be nice if the midi notes features could be toggled real time.
EG when the button is pressed then the keys on the keyboard are used to send messages like a sift or ctrl key on a pc keyboard.
This is the way my edirol PCR M30 works implicitly.  It has all the commands written out above the keys, so when I hit edit mode, I can relative easily reconfigure the device.

**Comment by Banana71** [comment]
Hi "RK with five 11 and 1", 

Do I understand correctly, you would like a switch (MIDI CC) that enables the MIDIButtonNotes navigation function? This would especially make sense for small MIDI keyboards. 👍

**Comment by probonopd** [comment]
Kinda like a shift button but for the MIDI keyboard?

**Reply by RK11111111111** [reply]
I think of it more like a Ctrl or Function key, but yeah that is the idea.  I would want to use the other buttons as shift keys, maybe to switch legato mode or something while playing.
